### PR TITLE
Add helper module for interaction with xmpp roster

### PR DESCRIPTION
**Pull request by sabudaye**
First, thank you for a great work you did! This library is exactly what i need, and works really good. But i think, in cases of roster management, it will be great to have some helper module, which encapsulates logic of parsing server xml response.

### CODE REVIEW

File: lib/romeo/roster/item.ex [hunk 1]
```diff
@@ -0,0 +1,23 @@
+defmodule Item do
```
Comment (scrogson) [inline lib/romeo/roster/item.ex]: This should be `Romeo.Roster.Item`

File: lib/romeo/roster/item.ex [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  use Romeo.XML
+
+  alias Romeo.Item
```
Comment (scrogson) [inline lib/romeo/roster/item.ex]: This is unused.

File: lib/romeo/roster/item.ex [hunk 3]
```diff
@@ -0,0 +1,23 @@
+defmodule Item do
+  use Romeo.XML
```
Comment (scrogson) [inline lib/romeo/roster/item.ex]: This is unused.

File: lib/romeo/stanza/attribute_parser.ex [hunk 4]
```diff
@@ -0,0 +1,21 @@
+defmodule Romeo.Stanza.AttributeParser do
```
Comment (scrogson) [inline lib/romeo/stanza/attribute_parser.ex]: Let's keep this in the parser module please.

File: mix.lock [hunk 5]
```diff
@@ -2,18 +2,18 @@
   "certifi": {:hex, :certifi, "0.4.0", "a7966efb868b179023618d29a407548f70c52466bf1849b9e8ebd0e34b7ea11f", [:rebar3], []},
```
Comment (scrogson) [inline mix.lock:2]: can you remove this file from this PR?

File: lib/romeo/stanza.ex [hunk 6]
```diff
@@ -151,6 +151,28 @@ defmodule Romeo.Stanza do
     iq("get", xmlel(name: "query", attrs: [{"xmlns", ns_roster}]))
   end
 
+  def set_to_roster(jid, subscription \\ "both", name \\ "", group \\ "") do
```
Comment (scrogson) [inline lib/romeo/stanza.ex]: Not a big fan of this name. What about `set_roster_item`?

File: lib/romeo/stanza/roster/parser.ex [hunk 7]
```diff
@@ -0,0 +1,25 @@
+defmodule Romeo.Stanza.Roster.Parser do
```
Comment (scrogson) [inline lib/romeo/stanza/roster/parser.ex]: I would just add this to `Romeo.Stanza.Parser`.

File: lib/romeo/roster.ex [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    case send_stanza(pid, stanza) do
+      {:ok, _} -> :ok
+      _ -> :error
+    end
+  end
+
+  defp send_stanza(pid, stanza) do
+    :ok = Conn.send(pid, stanza)
+    receive do
+      {:stanza, %IQ{type: "result"} = result} -> {:ok, result}
```
Comment (scrogson) [inline lib/romeo/roster.ex]: In order to ensure that the message received back from the server as a result for this request is to pattern-match the `id` field from the stanza.

**Comment by scrogson** [comment]
Hi @sabudaye, thank you for taking the time to put this PR together. I've added some comments.

**Comment by sabudaye** [comment]
Hi @scrogson , thank you for review, I made changes you requested. Please let me know if there are any remarks

**Comment by scrogson** [comment]
👍 Excellent! Thank you very much!

**Pull request merged**